WordPressKit/SharingServiceRemote.swift

```diff
@@ -44,50 +44,49 @@ open class SharingServiceRemote: ServiceRemoteWordPressComREST {
         let path = self.path(forEndpoint: endpoint, withVersion: ._1_1)
         let params = ["type": "publicize"]
 
-        wordPressComRestApi.GET(path,
-            parameters: params as [String: AnyObject]?,
-            success: { (responseObject: AnyObject, httpResponse: HTTPURLResponse?) in
-                guard let onSuccess = success else {
-                    return
-                }
+        wordPressComRestApi.GET(path, parameters: params as [String: AnyObject]?) { responseObject, httpResponse in
+            guard let onSuccess = success else {
+                return
+            }
```

Comment:
This `guard` seems unnecessary to me. It makes the `success` block required here in order to hit the failure `guard` case right after it.

Since it's the older version and hasn't caused any issues, I'll leave it up to you if you want to remove it.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/600/commits/62aa153633cfdbe5d6a61a4ade6b62876e2da087 ✅